File: Selenium/Project/Activity6.java
package project;

import org.testng.annotations.Test;

import io.github.bonigarcia.wdm.WebDriverManager;

import org.testng.annotations.BeforeClass;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;

public class Activity6 {
	
	WebDriver driver;
	
	  @BeforeClass
	  public void beforeClass() {
		  
		  WebDriverManager.firefoxdriver().setup();
		  driver = new FirefoxDriver();
		  driver.get("http://alchemy.hguy.co/orangehrm");
	  }
  @Test
  public void dirMenu() {
	  
	  	WebElement username = driver.findElement(By.id("txtUsername"));
		WebElement password = driver.findElement(By.id("txtPassword"));
		username.sendKeys("orange");
		password.sendKeys("orangepassword123");
		WebElement loginButton = driver.findElement(By.name("Submit"));
		loginButton.click();
		driver.findElement(By.xpath("//a[@href='/orangehrm/symfony/web/index.php/directory/viewDirectory/reset/1']")).click();
		WebElement header = driver.findElement(By.tagName("h1"));
		System.out.println("Title of the page is : " + header.getText());
		Assert.assertEquals(header.getText(), "Search Directory");
  }
  
  @AfterClass
  public void afterClass() {
	  
	  driver.close();
  }

}

File: Selenium/Project/activity6.py
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains
from selenium.webdriver.support.ui import Select

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver :
    Action = ActionChains(driver)
    driver.get("http://alchemy.hguy.co/orangehrm")

    username = driver.find_element(By.ID,"txtUsername")
    password = driver.find_element(By.ID,"txtPassword")

    username.send_keys("orange")
    password.send_keys("orangepassword123")

    loginButton = driver.find_element(By.NAME,"Submit")
    loginButton.click()

driver.find_element(By.XPATH,"//a[@href='/orangehrm/symfony/web/index.php/directory/viewDirectory/reset/1']").click()

header = driver.find_element(By.TAG_NAME,"h1")
actualTitle = "Search Directory"
print("Title of the page is : " , header.text)
assert actualTitle == header.text, "Title match"
driver.close()